src/plugins/dashboard/public/application/utils/get_table_columns.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { History } from 'history';
+import { EuiLink } from '@elastic/eui';
+import { i18n } from '@osd/i18n';
+import { ApplicationStart } from 'opensearch-dashboards/public';
+import { IUiSettingsClient } from 'src/core/public';
+import moment from 'moment';
+
+export const getTableColumns = (
```

Comment:
why'd you add this under utils? It seems pretty specific to the `dashboard_listing` - do you have plans to use it elsewhere?

Reply:
answered in a comment below~